_posts/2021-07-19-changelog-151.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #151"
+date: 2021-07-19 12:00:00 +0300
+---
+
+
+## New Features
+
+* [#6392] Add `Unused import` inspection. Note, this inspection is currently disabled by default due to possible false-positive warnings.
+  You can enable it via `Preferences | Editor | Inspections` [settings](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-analysis.html#inspection-settings) (by [@Kobzol])
+    <!-- Maybe get rid of these two items? -->
+    * [#7507] Support declarative macros
+
+    * [#7461] Correctly handle `writeln!` macro
+
+* [#7432] Confirm loading untrusted Cargo project
+  <!-- TODO: more information? -->
```

Comment:
Let's add a reason why we need it at all and link to platform help article https://www.jetbrains.com/help/idea/project-security.html